generate/convert.go

```diff
@@ -303,8 +303,23 @@ func (g *generator) convertDefinition(
 	// want, subject to the caveats described in Config.Bindings.)  Local
 	// bindings are checked in the caller (convertType) and never get here,
 	// unless the binding is "-" which means "ignore the global binding".
-	globalBinding, ok := g.Config.Bindings[def.Name]
-	if ok && options.Bind != "-" {
+	var (
+		hasBinding    bool
```

Comment:
Why do we need `hasBinding`? What if we don't have this variable at all?

In what scenario will that be a problem?